### PR TITLE
Add more versions to the changelog.

### PR DESCRIPTION
**Pull request by morenol**
In this PR I added description to the changelog for some of the last releases of this project.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -21,3 +21,51 @@ Added
...
+
+* Adding sentry integration
+
+[2.6.0] - 2020-01-09
+--------------------
+
+Removed
+~~~~~~~
+
+* Remove microsite configuration mentions.
```
Comment (Squirrel18) [inline CHANGELOG.rst:71]: @morenol  Could you please add this release:
0.14.0
Added
Course management automation.
Description
This new Studio module allows you to make changes to the course configuration for several courses at once.
More information: https://github.com/eduNEXT/eox-core#course-management-automation
Added
Linting tests (pylint - eslint) to CircleCI test suites.

Comment (morenol) [reply CHANGELOG.rst:71]: done

Comment (Squirrel18) [reply CHANGELOG.rst:71]: Thanks.

**Comment by Squirrel18** [comment]
Thanks @morenol squash and rebase. Maybe @felipemontoya or @andrey-canon or @jfavellar90  could help add more changelog versions to the file.

**Pull request merged**